src/components/Shop/Shop.js

```diff
@@ -135,6 +139,23 @@ export const Shop = ({
           }}
         />
       </li>
+      {features.MINING ? (
+        <li>
+          <TierPurchase
+            {...{
+              description:
+                'You can purchase a Smelter to convert ore into ingots, and other useful items.',
```

Comment:
Grammar nit: Remove the comma and change this to:

```
'You can purchase a Smelter to convert ore into ingots and other useful items.'
```

(I'm trying to break myself of this habit too!)

Reply:
ah yeah i get comma happy hah